test/support/annotations/vcr_request.rb

```diff
@@ -0,0 +1,34 @@
+module Annotations
+  class VcrRequest
+    attr_accessor :method, :path, :request_body, :response_body, :response_code
```

Comment:
Minor point, but looking at the code I think most if not all of these could just be `attr_reader` for outside access.